src/opl.c

```diff
@@ -1681,6 +1735,17 @@ int main(int argc, char *argv[])
 
     // reset, load modules
     reset();
+
+    /* argv[0] boot path
+       argv[1] game->startup
+       argv[2] str to u32 game->start_sector
+       argv[3] opl partition read from hdd0:__common/OPL/conf_hdd.cfg
+       argv[4] "mini" (this is probably unnecessary.. how many args would there be typically?) */
```

Comment:
As I understand we are forced to use https://github.com/ps2homebrew/OPL-Launcher ? Maybe it is really better than using opl itself.

Reply:
I don’t understand what you mean. 

OPL launcher does not launch games like miniOPL does, all it does is read the game partition and send the info as args to OPL.

If OPL boots up with like 5 args it will run the auto boot function with the info received from OPL launcher.

The whole purpose being that users still have that miniOPL feel of auto launching games from HDDOSD without ever having to worry about updating miniOPL since it will now work with all (future) versions of OPL.

Anyone else can still boot OPL in other ways, wLE or whatever and it will just go to the GUI since it’s not going to be sending 5 args with it.

Reply:
`Maybe it is really better than using opl itself.`

I also don’t understand why you’re being a sarcastic d*ck, if you can’t read the code maybe just don’t say anything.. you are one of the people that wanted this which is why I wasted my time on it... you’re welcome.